rest/route/test.go

```diff
@@ -69,6 +69,8 @@ func (tgh *testGetHandler) Parse(ctx context.Context, r *http.Request) error {
 				StatusCode: http.StatusBadRequest,
 			}
 		}
+	} else {
+		tgh.testExecution = projCtx.Task.Execution
```

Comment:
is projCtx.Task always the latest task?

Reply:
yeah i think all the older ones get archived so the one we do get should always be the latest execution

Reply:
true